source/OptimaJet.DWKit.StarterApplication/Models/Product.cs

```diff
@@ -42,5 +43,8 @@ public class Product : Identifiable, ITrackDate
         [Attr("date-published")]
         public DateTime? DatePublished { get; set; }
 
+        [HasMany("product-artifacts", Link.None)]
```

Comment:
This could be just "artifacts" for the json api name and "Artifacts" for the C# name.  Not a big deal.